tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -85,15 +85,15 @@ x
         )
 #endif
 
-    [<Fact>]
+    [<Fact(Skip="TBD")>]
```

Comment:
The logic of this test is unclear to me. The stream that the executed `Console.Readline` reads from is not necessarily the same as the one we passed to the `FsiEvaluationSession.Create`. We're just testing `Console.Readline` here. We probably should be testing that `FsiEvaluationSession` reads from the given input stream.

Reply:
Yeah, this looks like yet another weird test, thanks for noticing :)